# imavirus: finding virus integration sites, and more!* (*allegedly)


## Contributors

Alejandro Gener (lead) 
Michael Jochum 
Hiroko Ohmiya 
Carolina Peralta 
Remy Schwab 
Rajarshi Mondal 
Daniel Cameron 
Enrico Barrozo 
Fawaz Dabbaghie 


## Description
Identifying insertions and in particular virus integration sites is challenging. Moreover, there are not many tools to reliabliy detect viral presence and integration events. 
Many virus have the capability of integrating into host genomes and can lead to DNA damage and disrupt genes. 	
The aim of this project is to find putative virus integration sites (pIS) in public data. Identifying insertion sites from genomic DNA is challenging.


## Goals

This project has two main aims: 1) to identify (putative) integration sites (pIS) in public data and 2) Determining whether insertion sites occur at sites which may affect cell-function (contribute to disease?) and/or anti-viral responses (which may contribute to virus fitness). 


## Overview Diagram

![](https://github.com/collaborativebioinformatics/imavirus/blob/2f27cd88604fc01b134df44fc492afca9830ef25/Untitled%20Diagram%20v2.jpg)

## Test data

![](https://github.com/collaborativebioinformatics/imavirus/blob/faea5562d55e5ac4700ae02ffdca48d0b5d2b284/Screen%20Shot%202021-10-12%20at%2011.51.39%20AM.png)

Above depends on depositors including virus-specific metadata.

For EBV, ENCODE data from lymphoblasrtoid cell lines may not have EBV metadata.

## Results
